centaur/src/main/resources/standardTestCases/papi_v2_gcsa.test

```diff
@@ -0,0 +1,17 @@
+name: papi_v2_gcsa
+testFormat: WorkflowSuccess
+backends: [papi-v2-gcsa]
+
+files {
+  workflow: papi_v2_gcsa/papi_v2_gcsa.wdl
+  options-dir: ../../../../../target/ci/resources/
```

Comment:
Duplicate key possibly left over from local development?

Reply:
> Duplicate key possibly left over from local development?

The line below optionally overrides this line.

The override is for folks that use `src/ci/bin/testCentaurBackend.sh` for local testing. Those `bin` scripts are the most comprehensive ways of testing locally and mostly match CI on Travis/Jenkins. They render secrets from vault, (optionally) assemble the cromwell jar, and even run restart tests that start/stop cromwell. The require a bit to setup but after that they just work™.

However for simpler centaur tests devs sometimes spin up cromwell+centaur quickly using IntelliJ. In that case the environment variable on the _next line_ won't have been initialized, but the rendered json can still be found in the target directory on _this line_.

All of this convoluted logic is to avoid dropping rendered secrets file in our source directories, where there are likely to get (and have already gotten) swept up in a `git add . && git commit && git push`. Not a huge deal when caught immediately, we just have to rotate the respective creds in vault, but it's a PITA.

I plan on banishing other cases of generated-secret-into-source-code [[0]](https://github.com/broadinstitute/cromwell/blob/44/centaur/src/main/resources/standardTestCases/docker_hash_dockerhub_private_usa.test#L9) [[1]](https://github.com/broadinstitute/cromwell/blob/44/centaur/src/main/resources/standardTestCases/docker_hash_dockerhub_private_wf_options.test#L9) over into target directories, avoiding accidental secret commits. But I'll leave that for a future PR.

Side note, one day git-secrets will stop being a "Well, it's _your_ fault it's so easy to screw up and give you a false sense of confidence..." [[2]](http://www.dereferer.org/?https%3A%2F%2Fgithub%2Ecom%2Fawslabs%2Fgit-secrets%2Fissues%2F47) [[3]](http://www.dereferer.org/?https%3A%2F%2Fgithub%2Ecom%2Fawslabs%2Fgit-secrets%2Fissues%2F79) [[4]](https://stackoverflow.com/questions/22100581/github-server-side-git-hooks-i-e-pre-commit-and-post-commit)